exchange/localizations/localization.py

```diff
@@ -0,0 +1,19 @@
+import os
+import json
+
+from ..meta import SingletoneMeta
```

Comment:
Лучше избегать `..` при импортах

```suggestion
from exchange.meta import SingletoneMeta
```

Reply:
исправил